### DISCUSSION TITLE
Support `font-lock-face` in `corfu-popupinfo`

### DISCUSSION THREAD

**Discussion by jdtsmith** [minad/corfu#483] [Ideas]
Some docs come with styling based on `font-lock-face`, e.g. those with translated ansi-color codes.  It would be great if the popupinfo buffer supported display of `font-lock-face`.  Doing so is quite simple: just enable the alias among the parameters set in the hidden buffer:

```diff
diff -u [0] [9]
--- <corfu-popupinfo.el> [mod 0] (Marked)	2024-04-24 01:06:52 PM
+++ <corfu-popupinfo.el> [mod 9] (Current)	2024-07-16 03:55:59 PM
@@ -130,7 +130,8 @@
     (left-margin-width . 1)
     (right-margin-width . 1)
     (word-wrap . t)
-    (fringe-indicator-alist (continuation)))
+    (fringe-indicator-alist (continuation))
+    (char-property-alias-alist (face font-lock-face)))
   "Buffer parameters.")
 
 (defvar corfu-popupinfo--frame nil
```

Alternatively you could enable `font-lock` in that buffer (which adds this alias when turned on).

**Comment by minad** [comment]
JD Smith ***@***.***> writes:

> Some docs come with styling based on font-lock-face, e.g. those processed
> through ansi-color codes. It would be great if the popupinfo buffer supported
> display font-lock-face. Doing so is quite simple: just enable the alias among the
> parameters set in the hidden buffer:

Good idea. Pushed.
